core/storage/collection/gae_models.py

```diff
@@ -424,6 +447,11 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.KEEP_IF_PUBLIC
 
+    @staticmethod
+    def get_export_policy():
```

Comment:
Won't it have the ID of the committer?

Reply:
It seems like a lot of the models that were not outlined in the takeout document are ones that export from base model. Should I just place the export policy in the base model? I think just because the ID of the committer is contained, that doesn't necessarily mean that its important data.

Reply:
I'm not sure why this was resolved. It doesn't seem to have been addressed. Do you recall what the outcome of this discussion was?

In particular, do these models have the ID of the committer?

Reply:
I think this does contain user data. For the sake of this PR, I'm going to move the model into the TO_BE_IMPLEMENTED